configs/mysql/templates/psp.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+  name: mysql-psp
+spec:
+  privileged: true
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - emptyDir
+  - secret
```

Comment:
How about we just add a rolebinding to bind the privileged PSP to the required SA? Something like this: https://github.com/kinvolk/service-mesh-benchmark/blob/86f6ab99532b0c274f0f91de2e45f8e298b3f426/configs/orchestrator/templates/rbac.yaml#L63-L81